simulariumio/orientations/rotation_utility.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from sys import float_info
+import math
+
+import numpy as np
+from scipy.linalg import expm
+from scipy.spatial.transform import Rotation
+import random
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class RotationUtility:
```

Comment:
does this really need to be a class?  It could just be a plain old Python module and the function calls namespaced to the import itself.

Reply:
It makes imports nicer? so I don't have to export all the functions that are available in __init__.py. Is there a nice way to do that without a class?

Reply:
I think it's something like `from rotation_utility import *` or just `import rotation_utility` to maintain the module namespacing like I wrote above.

Reply:
I know I could do `from rotation_utility import *` but then it can be hard to tell where the functions came from, I tried just `import rotation_utility` and maybe I'm doing that wrong but can't get it to work... so I think I'm going to leave it as a class.